modules/cram/samtools.nf

```diff
@@ -8,3 +8,16 @@ process samtools_index {
 
     template 'samtools_index.sh'
 }
+
+process samtools_addreplacerg
```

Comment:
what does this process do? how about adding a bit of doc to describe?

Reply:
check: the comment is currently at the place its called in the cram workflow

Reply:
fixed on line above, therefor not picked up by github